### PR TITLE
Refactor Airbrake.blacklist/whitelist_keys to be an option

### LINKED ISSUE #53: Expose blacklist_keys and whitelist_keys as config options
I just realised it makes no sense to have those as methods.

Therefore this:

```ruby
Airbrake.blacklist_keys([:user, :password, :credit_card])
```

Should become this:

```ruby
Airbrake.configure do |config|
  config.blacklist_keys = [:user, :password, :credit_card]
end
```

@vmihailenco thoughts?

### PR DESCRIPTION
**Pull request by kyrylo**
Fixes #53 (Expose blacklist_keys and whitelist_keys as config options)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -258,6 +258,63 @@ Airbrake.configure do |c|
 end
 ```
 
+#### blacklist_keys
+
+Specifies which keys in the payload (parameters, session data, environment data,
+etc) should be filtered. Before sending an error, filtered keys will be
+substituted with the `[Filtered]` label.
+
+It accepts Strings, Symbols & Regexps, which represent keys to be filtered.
```
Comment (shifi) [inline README.md]: Nitpick: values are filtered here, so maybe change this to be more consistent with L291

File: lib/airbrake-ruby/config.rb [hunk 2]
```diff
@@ -58,6 +58,18 @@ class Config
     attr_accessor :timeout
 
     ##
+    # @return [Array<String, Symbol, Regexp>] keys The keys, which should be
```
Comment (shifi) [inline lib/airbrake-ruby/config.rb]: extra `keys`?

File: lib/airbrake-ruby/notifier.rb [hunk 3]
```diff
@@ -60,13 +73,23 @@ def add_filter(filter = nil, &block)
...
+      )
       add_filter(Filters::KeysWhitelist.new(*keys))
     end
 
     ##
     # @macro see_public_api_method
+    # @deprecated Please use {Airbrake::Config#blacklist_keys} instead
     def blacklist_keys(keys)
+      @config.logger.warn(
+        "#{LOG_LABEL} Airbrake.whitelist_keys is deprecated. Please use the " \
```
Comment (shifi) [inline lib/airbrake-ruby/notifier.rb]: `whitelist_keys` -> `blacklist_keys`

**Comment by shifi** [comment]
Few minor comments, otherwise LGTM

File: README.md (hunk 1 above)
Comment (kyrylo) [inline README.md]: Changed to `It accepts Strings, Symbols & Regexps, which represent keys of values to be
filtered.`, thanks!

File: lib/airbrake-ruby/config.rb (hunk 2 above)
Comment (kyrylo) [inline lib/airbrake-ruby/config.rb]: Yeah, copy-pasta :spaghetti:. Fixed.

File: lib/airbrake-ruby/notifier.rb (hunk 3 above)
Comment (kyrylo) [inline lib/airbrake-ruby/notifier.rb]: Fixed.

**Pull request merged**